cypress/e2e/work-order/attend/close-with-followons-enabled.cy.js

```diff
@@ -430,6 +416,8 @@ describe('Closing my own work order - When follow-ons are enabled', () => {
     })
 
     it('payment type selection is not possible, closing makes a POST request for no access with bonus payment type, confirms success, and returns me to the index', () => {
+      interceptFileUpload()
+
```

Comment:
Given photo upload is mendatory, all tests will select a photo and intercept the requests